mysql-py/mysql-conn.py

```diff
@@ -1,22 +1,17 @@
 #!/usr/bin/python
-
 hostname = '127.0.0.1'
```

Comment:
Can you make this parameters dynamic ? Use a getopts like tool to parse arguments.